EIPS/eip-5083.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 5083
+title: NFT Token Acceptance
```

Comment:
Reading the title and description I feel the "token acceptance" is a bit ambiguous. Can you reduce the ambiguity in the title to match the description?